README.md

```diff
@@ -1,43 +1,99 @@
-# Mattermost LLM Extensions Plugin
+# Mattermost AI Plugin
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+![Screenshot](/img/mention_bot.png)
+
+## Table of Contents
+
+- [Mattermost AI Plugin](#mattermost-ai-plugin)
+  - [Table of Contents](#table-of-contents)
+  - [Background](#background)
+  - [Install](#install)
+  - [Usage](#usage)
+    - [Conversation](#conversation)
+    - [Thread Summarization](#thread-summarization)
+    - [Answer questions about Threads](#answer-questions-about-threads)
+    - [Chat anywhere](#chat-anywhere)
+    - [React for me](#react-for-me)
+    - [RLHF Feedback Collection](#rlhf-feedback-collection)
+  - [Related Efforts](#related-efforts)
+  - [Contributing](#contributing)
+  - [License](#license)
+
+## Background
+
+**🚀 Check out [our AI developer website](https://mattermost.github.io/mattermost-ai-site/) and join the ["AI Exchange" channel](https://community.mattermost.com/core/channels/ai-exchange) where Mattermost's open source community is sharing AI news and innovation in real time!**
 
 The LLM extensions plugin adds functionality around the use and development of LLMs like GPT-3.5 / 4 and hugging face models within Mattermost. 
 
 Currently at the experimental phase of development. Contributions and suggestions welcome! 
 
-## Conversation
+## Install
+
+1. Clone and enter this repository:
+  * `git clone https://github.com/mattermost/mattermost-plugin-ai && cd mattermost-plugin-ai`
+2. Start the services: `docker compose up -d`
+3. Configure the Mattermost server from the init script: `bash ./init.sh`
+4. Access Mattermost:
+  * Open Mattermost at `http://localhost:8065`
+  * Select **View in Browser**
+  * Log in with the generated `root` credentials
+5. Install the `mattermost-plugin-ai` on Mattermost:
+  * Go the releases page and download the latest release.
```

Comment:
Don't actually have releases yet. I can add one if you would like. Or maybe just run `make dist` for the quick setup for now?

Reply:
Oops, looks like my final README version with `make deploy` didn't make the PR. I just pushed it now.